contrib/examples/delete-annotations-simple/README.md

```diff
@@ -0,0 +1,57 @@
+# delete-annotations: Simple Example
+
+### Overview
+
+In this example, we will see how to delete annotations on a set of resources in a package/folder
+
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/contrib/examples/delete-annotations-simple
+```
+
+### Function invocation
+
+Invoke the function by running the following command:
+
+```shell
+$ kpt fn render delete-annotations-simple
+```
+
+### Expected result
+
+One of the two resources in `resources.yaml` should have been mutated with the annotation `annotation.to.delete` removed from `metadata.annotations`. There shouldn't be any changes to the second resource as it doesn't have the supplied annotation.
+
+```yaml
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sRestrictRoleBindings
```

Comment:
I guess we don't need to copy the output here, since what we do here is not complex and static content here may get out-of-date.

Reply:
Removed.